test/mocks/mock_firebase_platform.dart

```diff
@@ -0,0 +1,4 @@
+import 'package:firebase_core_platform_interface/firebase_core_platform_interface.dart';
+import 'package:mockito/mockito.dart';
+
+class MockFirebasePlatform extends Mock implements FirebasePlatform {}
```

Comment:
I think it'd be great if we both got into the habit of commenting the classes we make in the mocks files - could you add a comment here?